### ISSUE TITLE
Cyber Engine UI is invisible, only showing mouse cursor. Can't click anything in game UI

### ISSUE THREAD

**Issue by orangpelupa** [yamashi/CyberEngineTweaks#474]
v1.10.2: 

Launching cyberpunk 2077 will bring me a mouse cursor that cannot be used to click anything in the game UI. Cyber Engine UI is not visible at all on screen.

I assume its waiting for me to set up the key binding.

If i manually set the hotkey thru the config file, i can play the game just fine. But pressing the hotkey will simply show a mouse cursor, cyber engine UI still invisible.

**Comment by yamashi** [comment]
Are you using overlays such as RivaTunrer?

**Comment by pchristod** [comment]
I had the exact same problem. And as @yamashi says it's due to an overlay. I'm using the standalone version of RivaTuner Statistics Server. It will work if you disable it.

**Comment by orangpelupa** [comment]
> Are you using overlays such as RivaTunrer?
Thank you
I do have RTSS running in background but with overlays disabled. 
Completely exiting RTSS makes Cyber Engine UI visible.

**Comment by JulianVolodia** [comment]
@orangpelupa - but if restarted it works? does '=' mapped to anything?

**Comment by WSSDude420** [comment]
@orangpelupa did you save that bind? if you bind, it should turn yellow.
pressing save should save those changes.

that issue seems odd, first time hearing it... would imagine it is this...

we should add some popup cause user accidentally closing overlay or switching tabs can will invalidate unsaved changes as of right now, which may lead to this situation.

**Comment by WSSDude420** [comment]
I will close this issue for now. If you require some further assistance, feel free to reopen.

**Comment by orangpelupa** [comment]
> @orangpelupa did you save that bind? if you bind, it should turn yellow.
> pressing save should save those changes and color should change to white.
> 
> that issue seems odd, first time hearing it... would imagine it is this...
> 
> we should add some popup cause user accidentally closing overlay or switching tabs will invalidate unsaved changes as of right now, which may lead to this situation.

Yeah the key binding is saved as I no longer need to rebind after cyberpunk relaunch. 

Maybe related to a bug where sometimes alt tabbing (or cyberpunk automatically goes minimized at launch) will make the game got "frozen frame"?

Anyway the Workaround of disabling RTSS and not pressing alt tab does work

**Comment by WSSDude420** [comment]
Ok so this is not yet resolved. 

I hopes as you were not answering that maybe you found out how to make it work. 

That is not ideal still tbh, would like to debug it... Are you on our community Discord by a chance? would be faster to solve it from there.

**Comment by orangpelupa** [comment]
> Ok so this is not yet resolved.
> 
> I hoped as you were not answering that maybe you found out how to make it work.
> 
> That is not ideal still tbh, would like to debug it... Are you on our community Discord by a chance? would be faster to solve it from there.
> 
> We would add some write-up here later.

Thank you. Sorry these past few days I've been overwhelmed with extra work. I'm not in the discord (actually I don't know this project have a discord haha) 

Anyway, let me know if you need me to do something. I hope I could get some free time this weekend.
